rbac/internal/urls.py

```diff
@@ -16,14 +16,28 @@
 #
 
 """Describes the urls and patterns for internal routes."""
+
 from django.urls import path
 
 from . import views
 
+
+class SentryDiagnosticError(Exception):
+    """Raise this to create an event in Sentry."""
+
+    pass
+
+
+def trigger_error(request):
```

Comment:
Would it make sense to move this into `/rbac/internal/views.py` instead of having it directly in the urls file?